ocean_provider/file_types/definitions.py

```diff
@@ -78,6 +82,25 @@ def check_details(self, with_checksum=False):
                 content_type = headers.get("Content-Type")
                 content_length = headers.get("Content-Length")
                 content_range = headers.get("Content-Range")
+                file_name = None
+
+                if files_url:
+                    filename = urlparse(files_url).path.split("/")[-1]
+                    try:
+                        if not self._validate_filename(filename):
+                            logger.error(
+                                "Invalid file name format. It was not possible to get the file name."
```

Comment:
to avoid duplicating this string when changing it, set it up as a variable e.g. msg="Invalid file format...", then logger.error(msg) and return False, {"err_msg": msg}. Also, can you please make it "error" instead of "err_msg"? It should look like the other errors to allow frontend to make their logic based on error messages. And please document it in the readme where we explain all the error formats.